prod/westeurope/internal/api/functions_assets_r3/function_app/terragrunt.hcl

```diff
@@ -90,14 +90,13 @@ inputs = {
 
     SLOT_TASK_HUBNAME = "ProductionTaskHub"
 
-    SERVICES_BY_SCOPE_BLOB_ID = "visible-services-by-scope.json"
-
   }
 
   app_settings_secrets = {
     key_vault_id = dependency.key_vault.outputs.id
     map = {
-
+      STATIC_WEB_ASSETS_ENDPOINT=common-STATIC-WEB-ASSETS-ENDPOINT
```

Comment:
can you format the code using `hclfmt`?

https://terragrunt.gruntwork.io/docs/getting-started/configuration/#formatting-terragrunthcl

Reply:
Fixed in [9c4f0c8](https://github.com/pagopa/io-infrastructure-live-new/pull/264/commits/9c4f0c8b98eadbd9f98b89dfa09e02208d9994f5)